packages/frontend/@n8n/design-system/specification/COMPONENT_API_SPEC_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+# Component specification
+
+Copy this file next to the template and call it `component-<component-name-kebap-case>.md`.
+
+This component specification describes the public API of a component intended to be added to the component library of our design system.
+To be made available for review in a GitHub pull request before implementation.
+With two approvals, you can be quite sure that the API won't be challenged anymore on the implementation PR.
+
+- **Component Name:** N8nCheckbox
```

Comment:
We should clearly state which component we're migrating from to which Reka UI component
Like "migrating from" or "old": **Element+ Component:** [ElCheckbox](https://element-plus.org/en-US/component/checkbox)

"new": **Reka UI Component:** [Checkbox](https://reka-ui.com/docs/components/checkbox)